llvm/include/llvm/CodeGen/ValueTypes.td

```diff
@@ -338,6 +338,8 @@ def amdgpuBufferFatPointer : ValueType<160, 234>;
 // FIXME: Remove this and the getPointerType() override if MVT::i82 is added.
 def amdgpuBufferStridedPointer : ValueType<192, 235>;
 
+def vi8       : ValueType<8,  236>;  // 8-bit integer in FPR (AArch64)
```

Comment:
Would it be possible to use the existing `v1i8` ?

Reply:
I tried using `v1i8` first, but you need to map the value type to the register class with `addRegisterClass(MVT::vi8, &AArch64::FPR8RegClass)` and `v1i8` already has a meaning and uses within the backend (and is an illegal type, which is scalarized). So adding `addRegisterClass(MVT::v1i8, &AArch64::FPR8RegClass)` breaks existing lowerings.

Reply:
Basically, I added `vi8` so I could have a legal type for 8-bit value in FPR, since both `v1i8` and `i8` are illegal types.